web/packages/teleport/src/services/resources/resource.ts

```diff
@@ -55,17 +55,30 @@ class ResourceService {
       });
   }
 
-  fetchGithubConnectors(): Promise<{
+  async fetchGithubConnectors(): Promise<{
     defaultConnector: DefaultAuthConnector;
     connectors: Resource<'github'>[];
   }> {
-    return api.get(cfg.getGithubConnectorsUrl()).then(res => ({
-      defaultConnector: {
-        name: res.defaultConnectorName,
-        type: res.defaultConnectorType,
+    // MFA reuse needs to be allowed in case we need to fallback to another default connector
+    const challenge = await auth.getMfaChallenge({
+      scope: MfaChallengeScope.ADMIN_ACTION,
+      allowReuse: true,
+      isMfaRequiredRequest: {
+        admin_action: {},
```

Comment:
what if this fails? do we need some sort of try/catch and error attempt here? or is it handled in some `useAsync` elsewhere?

Reply:
the `useAsync` where this is called handles it